packages/playwright-core/src/server/frames.ts

```diff
@@ -1057,6 +1058,13 @@ export class Frame extends SdkObject {
     });
   }
 
+  async rafrafScreenshotElementWithProgress(progress: Progress, selector: string, options: ScreenshotOptions): Promise<Buffer> {
```

Comment:
Let's think about `rafrafScreenshotElementWithProgress(...., minTimeout: number | undefined)` and skip rafraf/timeout when minTimeout is undefined.